WebCompiler/Compile/Compilers.cs

```diff
@@ -15,9 +16,16 @@ public class Compilers
         private readonly Compiler place;
         private readonly Compiler cleanup;
         private readonly Compiler autoprefix;
+        private readonly List<string> _ignoreFiles;
```

Comment:
Nit: please keep the variable naming consistent with the rest of the file, i.e. without leading underscore.